### ISSUE TITLE
Handle or Move C Library Branches/Issues/PRs to emokit-c

### ISSUE THREAD

**Issue by ghost** [openyou/emokit-c#1]
How do you want to deal with these?

**Comment by qdot** [comment]
Yeah I was thinking about that /after/ I asked you to merge, heh. I'll handle doing the cleanup and bringing the usable patches and viable bugs over to the C repo. Most of the patches are going to require hand merging anyways. If you want to assign any C bug to me, that's fine.
